src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5130,7 +5130,7 @@
                                   import_list:namespace,...
 /langversion:&lt;number&gt;             Specify language version: 
                                   9|9.0|10|10.0|11|11.0|12|12.0|14|14.0|15|
-                                  15.0|default|latest
+                                  15.0|15.3|default|latest
```

Comment:
I added a test for `ToDisplayString`.
`GetErrorName` always includes a decimal, but `ToDisplayString` uses integral numbers for major versions.
Not sure which is best.